.dockerignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
Just an FYI, dockerignore only affects the building of the image, not the volume mounts